osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/firmware/enums/FirmwareTransferStatus.java

```diff
@@ -0,0 +1,41 @@
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.firmware.enums;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum FirmwareTransferStatus {
```

Comment:
Having an enum for the status is a good idea. I think it would be good to add a little documentation linking this enum to the DLMS documentation.

It looks like it would be better to change the names a bit to more closely reflect how they are in the DLMS Blue Book with the descripton of attribute 6 image_transfer_status of the Image Transfer interface class.
The current names match what is described in SMR 5.1 P2 as firmware upgrade states, but I don't why the terms are different from DLMS, and the ImageTransfer class in the GXF DLMS protocol adapter is not only used with M-Bus firmware upgrades.

Reply:
I noticed an ImageTransferStatus enum already exists:
org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.firmware.ImageTransfer.ImageTransferStatus
It looks like there is no need to add another one.